Documentation/deps-versions.md

```diff
@@ -0,0 +1,9 @@
+|  OCP Version |                                       alertmanager                                       |                                  kubeRbacProxy                                   |                                  kubeStateMetrics                                   |                                                kubernetesMetricsServer                                                 |                                 monitoringPlugin                                  |                                 nodeExporter                                  |                                  promLabelProxy                                  |                                 prometheus                                  |                                    prometheusAdapter                                    |                                  prometheusOperator                                  |                                 thanos                                  |
```

Comment:
I wonder if it wouldn't be more readable to have dedicated sections per minor?

## 4.16

| Component | Version |
|--------|--------|
| Alertmanager | 0.26.0 |
| kube-rbac-proxy | 0.15.0 |
| kube-state-metrics | 2.10.1 | 
| ... | ... | 

## 4.15

...

Reply:
You'd need to scroll down a lot for older versions and the unique table easier to generate...

Reply:
If we go with a single table (no strong opinion on that) can we have the newest version on top?

Reply:
newest versions are on top now.